src/main/java/uk/gov/indexer/dao/DBQueryDAO.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.indexer.dao;
+
+public interface DBQueryDAO {
```

Comment:
Sorry I thought about Closeable but didn't use it because i wanted to make it explicit at shutdown time that all database connections are closed.